<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выполнение</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Визуализация</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QWidget" name="widgetplot" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Ось X</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="xaxes_combobox">
              <item>
               <property name="text">
                <string>отсчеты</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>напряжение, В</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Ось Y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="yaxes_combobox">
              <item>
               <property name="text">
                <string>сопротивление, кОм</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>сопротивление, Ом</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>отсчеты АЦП</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ток, мкА</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ток, мА</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="graph_checkbox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Отображение</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="plot_type_combobox">
              <item>
               <property name="text">
                <string>линия</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>точки</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>звездочки</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_graph_settings">
              <property name="text">
               <string>Настройки</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Управление выполнением</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="button_start">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Запуск</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_pause">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Пауза</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_stop">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Остановка</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="exp_progress">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_total_count">
          <property name="text">
           <string>Осталось задач:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_mem_id">
          <property name="text">
           <string>ID мемристора:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
